<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/iLogos/logo_novafct.png" width="200">

# Departamento de Engenharia Mecânica e Industrial
## Mecânica Aplicada I 

## EQUILÍBRIO DA PARTÍCULA NO ESPAÇO

### Problema 6 <a name="P7"></a>

Na figura está representado um veio de transmissão, ligado por uma correia ao motor $M$. O momento gerado no motor é transmitido ao tambor $D$, obtendo-se forças de tração nos ramos da correia ligada a este tambor, $T =$ 300 N, $T'$ = 180 N. As chumaceiras $A$ e $C$ apenas restringem movimentos de translação na direção transversal ao eixo do veio, pois montam rolamentos com compensação angular.  A chumaceira $A$ impede também movimentos axiais do veio. Determine:

[a)](#A) O sentido de rotação do veio, considerando que o motor está a fornecer potência ao sistema.

[b)](#B) As forças de tração nos dois ramos da correia que liga o tambor $B$ ao motor $M$, sabendo que a componente segundo $z$ da reação em $A$, vale $R_\textrm{AZ}$ = 10 N.

[c)](#C) As reações nas duas chumaceiras.

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/Notebooks/Ch04/P6/MAI_Ch04_P6.png"
width="500">

### Resolução

#### a) O sentido de rotação do veio, considerando que o motor está a fornecer potência ao sistema. <a id="A"></a> ([_enunciatum_](#P7))

**Diagramas de corpo livre**

Considerem-se as forças externas identificadas nos diagramas de corpo livre do veio da figura:

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/Notebooks/Ch04/P6/MAI_Ch04_P6a.jpg"
width="500">

Para analisar qualitativamente o problema, considere-se a ação dos momentos que as forças exercem em relação ao eixo do veio. O eixo do veio é escolhido criteriosamante porque elimina automaticamente a ação das forças de reação nas chumaceiras $A$ e $C$.

Considere-se como enunciado que as forças de tração nos ramos da correia ligada ao tambor $D$ é tal que: $T > T'$. Os momentos deste par de forças em relação ao veio é expresso por (escolhendo o ponto $D$ pertencente ao veio como ponto para o cálculo dos momentos):

\begin{equation*}
M_\textrm{veio}^D = \vec{\lambda}_\textrm{veio} \cdot (
\vec{M}_D^{T}-\vec{M}_D^{T'}) =
\vec{\lambda}_\textrm{veio} \cdot [
\vec{r}_{D}\times(\vec{T}-\vec{T'})]
\end{equation*}

Para haver equilíbrio estático é necessário que as forças de tração instaladas nos ramos da correia ligada ao tambor $B$ anulem o momento $M_\textrm{veio}^D$:

\begin{equation*}
M_\textrm{veio}^B = \vec{\lambda}_\textrm{veio} \cdot (
\vec{M}_D^{T}-\vec{M}_D^{T'}) =
\vec{\lambda}_\textrm{veio} \cdot [
\vec{r}_{B}\times(\vec{F}-\vec{F'})]
\end{equation*}

Concluindo-se do equilíbrio a seguinte relação: $F’ > F$, uma vez que $T > T’$.

Para que haja transmissão de potência pelas correias é necessário que a força no ramo tenso seja superior à força que se desenvolve no ramo bambo:

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/Notebooks/Ch04/P6/MAI_Ch04_P6b.jpg"
width="500">

Um vez que $F’ > F$, sobre o ramo inferior da correia é aplicada uma força maior em relação à sentida no ramo superior (figura), o primeiro é portanto o ramo tenso e o segundo o ramo bambo. Logo conclui-se que a rotação do motor é no sentido anti-horário.

Pela análise dinâmica, é possível estabelecer uma relação entre as forças no ramo tenso e no ramo bambo da correia, contudo este estudo está fora do âmbito do equilíbrio estático aqui invocado.

_Experiência_

Faça-se a seguinte experiência. Coloque um elástico de cabelo entre os dedos indicadores, com pré-tensão. Fixe um dos dedos e rode o outro num sentido arbitrário. Qual dos ramos está tenso e qual está frouxo? Qual é o padrão de correspondência entre a intensidade dessas forças e o sentido de rotação imposto?

#### b) As forças de tração nos dois ramos da correia que liga o tambor B ao motor M, sabendo que a componente segundo $z$ da reacção em A, vale $R_\textrm{AZ}$ = 10 N. <a id="B"></a> ([_enunciatum_](#P7))

**Equilíbrio estático**

Para o equilíbrio estático é condições necessárias e suficientes que se verifiquem as seguintes equações:

\begin{equation*}
\begin{cases}
\sum\vec{F} = \vec{0}\\
\sum\vec{M}_A = \sum\vec{r}\times\vec{F} = \vec{0}
\end{cases}
\end{equation*}

Para a análise do equilíbrio dos momentos é necessária a escolha prévia de um ponto do sistema. É conveniente escolher o  ponto que elimine, à partida, o maior número de incógnitas em termos de forças  desconhecidas. Neste caso, é escolhido o desenvolvimento da expressão do sumatório dos momentos exercídos pelas força aplicada em torno do ponto $A$.

**Componentes cartesianas dos vetores das forças**

A lista das componentes cartesianas das forças aplicadas é:

* incógnitas: $\vec{R}_\textrm{A}; \vec{R}_\textrm{C}; \vec{F}$

    * ($R_\textrm{Ax}; R_\textrm{Ay}; R_\textrm{Cy};
    R_\textrm{Cz}; F;F'$) com $R_\textrm{Az} = 10$ N

Para analisar o equilíbrio estático comece-se por escrever os vetores das forças envolvidas:

\begin{equation*}
\begin{cases}
\vec{R}_\textrm{A} &= R_\textrm{Ax}\,\vec{\imath} +
R_\textrm{Ay}\,\vec{\jmath} + 10\,\vec{k}\\
\vec{R}_\textrm{C} &= R_\textrm{Cy}\,\vec{\jmath} +
R_\textrm{Cz}\,\vec{k}\\
\vec{T} &= 300\,\vec{k}\\
\vec{T'} &= 180\,\vec{k}\\
\vec{F} &= -F\sin\alpha \,\vec{\jmath} + F\cos\alpha \,\vec{k}\\
\vec{F'} &= F'\,\vec{k}\\
\end{cases}
\end{equation*}

In [61]:
import numpy as np
import sympy as sy
from sympy.solvers import solve
from sympy.physics.mechanics import ReferenceFrame, cross

rax, ray, rcy, rcz, f1, f2 = sy.symbols('rax ray rcy rcz f1 f2')

T1 = 300 # unit: N
T2 = 180
RAZ = 10
angB = np.deg2rad(15) # unit: radianos

S = ReferenceFrame('S') # ||x|
vecRA = rax*S.x + ray*S.y + RAZ*S.z; print('vecRA : ',vecRA)
vecRC = rcy*S.y + rcz*S.z; print('vecRC : ',vecRC)
vecT1 = T1*S.z; print('vecT1 : ',vecT1)
vecT2 = T2*S.z; print('vecT2 : ',vecT2)
vecF1 = -f1*np.sin(angB)*S.y + f1*np.cos(angB)*S.z
print('vecF1 : ',vecF1)
vecF2 = f2*S.z; print('vecF2 : ',vecF2)

vecRA :  rax*S.x + ray*S.y + 10*S.z
vecRC :  rcy*S.y + rcz*S.z
vecT1 :  300*S.z
vecT2 :  180*S.z
vecF1 :  - 0.258819045102521*f1*S.y + 0.965925826289068*f1*S.z
vecF2 :  f2*S.z



**Equilíbrio de forças**

\begin{equation*}
\bullet\quad\sum \vec{F} = \vec{0}~\Leftrightarrow~
\begin{cases}
\rightarrow(+)&x:&\sum F_x = 0 \\
\uparrow(+)&y:&\sum F_y = 0 \\
\swarrow(+)&z:&\sum F_z = 0 \\
\end{cases}
\end{equation*}

Desenvolvendo,

\begin{equation*}
\begin{cases}
\sum F_x = 0 \\
\sum F_y = 0 \\
\sum F_z = 0 \\
\end{cases}~\Leftrightarrow~
\begin{cases}
R_\textrm{Ax} = 0 \\
R_\textrm{Ay} + R_\textrm{Cy} - F\sin(15\frac{\pi}{180}) = 0 \\
10 + R_\textrm{Cz} + 300 + 180 + F\cos(15\frac{\pi}{180}) + F'= 0\\
\end{cases}
\end{equation*}

In [62]:
FX = rax; print('FX : ',FX)
FY = ray + rcy - f1*np.sin(angB); print('FY : ',FY)
FZ = RAZ + rcz + T1 + T2 + f1*np.cos(angB) + f2; print('FZ : ',FZ)

FX :  rax
FY :  -0.258819045102521*f1 + ray + rcy
FZ :  0.965925826289068*f1 + f2 + rcz + 490


**Equilíbrio de momentos**

\begin{equation*}
\bullet\quad\sum \vec{M}_A = \vec{0}~\Leftrightarrow~
\begin{cases}
\vec{k}\times\vec{\jmath} = \vec{\imath}~(+)&:&\sum M_{Ax} = 0 \\
\vec{k}\times\vec{\imath} = \vec{\jmath}~(+)&:&\sum M_{Ay} = 0 \\
\vec{\imath}\times\vec{\jmath} = \vec{k}~(+)&:&\sum M_{Az} = 0 \\
\end{cases}
\end{equation*}

Desenvolvendo,

\begin{equation*}
\begin{aligned}
\sum \vec{M}_A &=
\sum_{i} (\vec{r}_{\textrm{A}i} \times \vec{F}_{i}) \\
&= (\vec{r}_{\textrm{A}E} \times \vec{F})
+ (\vec{r}_{\textrm{A}G} \times \vec{F}')
+ (\vec{r}_{\textrm{A}C} \times \vec{R}_C)
+ (\vec{r}_{\textrm{A}H} \times \vec{T}_C)
+ (\vec{r}_{\textrm{A}I} \times \vec{T}')
= \vec{0}
\end{aligned}
\end{equation*}

Para determinar os vetores posição $\vec{r}_\textrm{AE}$ e $\vec{r}_\textrm{AG}$, é útil considerar os esquemas na figura:

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/Notebooks/Ch04/P6/MAI_Ch04_P6c.jpg"
width="500">

Escrevendo os vetores dos braços das forças no sistema de coordenadas escolhido:

\begin{equation*}
\begin{aligned}
\vec{r}_\textrm{AE} &= l_\textrm{AB}\,\vec{\imath} +
r_\textrm{B}\cos(\alpha_\textrm{E})\,\vec{\jmath} +
r_\textrm{B}\sin(\alpha_\textrm{E})\,\vec{k}\\
\vec{r}_\textrm{AG} &= l_\textrm{AB}\,\vec{\imath} - r_\textrm{B}\,\vec{\jmath}\\
\vec{r}_\textrm{AC} &= l_\textrm{AC}\,\vec{\imath}\\
\vec{r}_\textrm{AH} &= l_\textrm{AD}\,\vec{\imath} + r_\textrm{D}\,
\vec{\jmath}\\
\vec{r}_\textrm{AI} &= l_\textrm{AD}\,\vec{\imath} - r_\textrm{D}\,\vec{\jmath}
\end{aligned}
\end{equation*}

In [63]:
lAB = 0.3; lBC = lAB; lCD = lAB; print('lAB, lBC, lCD = ',lBC,' m')
lAC = lAB + lBC; print('lAC = ',lAC,' m')
lAD = lAB + lBC + lCD; print('lAD = ','{:.1f}'.format(lAD),' m')
rB = 0.1; print('rB = ',rB,' m')
rD = 0.15; print('rD = ',rD,' m')

rAE = lAB*S.x + rB*np.cos(angB)*S.y + rB*np.sin(angB)*S.z
print('rAE : ',rAE)
rAG = lAB*S.x - rB*S.y; print('rAG : ',rAG)
rAC = lAC*S.x; print('rAC : ',rAC)
rAH = lAD*S.x + rD*S.y; print('rAH : ',rAH)
rAI = lAD*S.x - rD*S.y; print('rAI : ',rAI)

lAB, lBC, lCD =  0.3  m
lAC =  0.6  m
lAD =  0.9  m
rB =  0.1  m
rD =  0.15  m
rAE :  0.3*S.x + 0.0965925826289068*S.y + 0.0258819045102521*S.z
rAG :  0.3*S.x - 0.1*S.y
rAC :  0.6*S.x
rAH :  0.9*S.x + 0.15*S.y
rAI :  0.9*S.x - 0.15*S.y



Numericamente,

\begin{equation*}
\begin{aligned}
\vec{r}_\textrm{AE} &= 0.1\,\vec{\imath} +
0.1\cos(15\frac{\pi}{180})\,\vec{\jmath} +
0.1\sin(15\frac{\pi}{180})\,\vec{k}\\
&= 0.1\,\vec{\imath} +
0.0966\,\vec{\jmath} +
0.0259\,\vec{k}\\
\vec{r}_\textrm{AG} &= 0.3\,\vec{\imath} - 0.1\,\vec{\jmath}\\
\vec{r}_\textrm{AC} &= 0.6\,\vec{\imath}\\
\vec{r}_\textrm{AH} &= 0.9\,\vec{\imath} + 0.15\,\vec{\jmath}\\
\vec{r}_\textrm{AI} &= 0.9\,\vec{\imath} - 0.15\,\vec{\jmath}
\end{aligned}
\end{equation*}

O cálculo do produto vetorial pode ser executado usando uma notação matricial, recorrendo ao seguinte determinante:

\begin{equation*}
\begin{aligned}
\sum \vec{M}_\textrm{A} &=
 (\vec{r}_\textrm{AE}\times\vec{F})
+ (\vec{r}_\textrm{AG}\times\vec{F}')
+ (\vec{r}_\textrm{AC}\times\vec{R}_\textrm{C})
+ (\vec{r}_\textrm{AH}\times\vec{T}_\textrm{C})
+ (\vec{r}_\textrm{AI}\times\vec{T}')
= \vec{0}
\end{aligned}
\end{equation*}

\begin{equation*}
\vec{r}_\textrm{AE}\times\vec{F} =
\begin{vmatrix}
\vec{\imath} & \vec{\jmath} & \vec{k} \\
x_{\textrm{AE}} & y_{\textrm{AE}} & z_{\textrm{AE}} \\
0 & -0.26\,F & 0.97\,F
\end{vmatrix} =
F(0.1\vec{\imath} - 0.290\vec{\jmath} - 0.078\vec{k})
\end{equation*}

In [64]:
CP1 = cross(rAE,vecF1); print('Cross 1: ',CP1)

Cross 1:  0.1*f1*S.x - 0.289777747886721*f1*S.y - 0.0776457135307562*f1*S.z



\begin{equation*}
\vec{r}_{\textrm{AG}}\times\vec{F}'=
\begin{vmatrix}
\vec{\imath} & \vec{\jmath} & \vec{k} \\
x_{\textrm{AG}} & y_{\textrm{AG}} & 0 \\
0 & 0 & F'
\end{vmatrix} =
F'(-0.1\vec{\imath} - 0.3\vec{\jmath})
\end{equation*}

In [65]:
CP2 = cross(rAG,vecF2); print('Cross 2: ',CP2)

Cross 2:  - 0.1*f2*S.x - 0.3*f2*S.y



\begin{equation*}
\vec{r}_{\textrm{AC}}\times\vec{R}_\textrm{C}=
\begin{vmatrix}
\vec{\imath} & \vec{\jmath} & \vec{k} \\
x_{\textrm{AC}} & 0 & 0 \\
0 & R_\textrm{Cy} & R_\textrm{Cz}
\end{vmatrix} =
-0.6R_\textrm{Cz}\vec{\jmath} + 0.6R_\textrm{Cy}\vec{k}
\end{equation*}

In [66]:
CP3 = cross(rAC,vecRC); print('Cross 3: ',CP3)

Cross 3:  - 0.6*rcz*S.y + 0.6*rcy*S.z



\begin{equation*}
\vec{r}_{\textrm{AH}}\times\vec{T}=
\begin{vmatrix}
\vec{\imath} & \vec{\jmath} & \vec{k} \\
x_{\textrm{AH}} & y_{\textrm{AH}} & 0 \\
0 & 0 & T
\end{vmatrix} =
45.0\vec{\imath} - 270.0\vec{\jmath}
\end{equation*}

In [67]:
CP4 = cross(rAH,vecT1); print('Cross 4: ',CP4)

Cross 4:  45.0*S.x - 270.0*S.y



\begin{equation*}
\vec{r}_{\textrm{AI}}\times\vec{T}'=
\begin{vmatrix}
\vec{\imath} & \vec{\jmath} & \vec{k} \\
x_{\textrm{AI}} & y_{\textrm{AI}} & 0 \\
0 & 0 & T'
\end{vmatrix} =
-27.0\vec{\imath} - 162.0\vec{\jmath}
\end{equation*}

In [68]:
CP5 = cross(rAI,vecT2); print('Cross 5: ',CP5)

Cross 5:  - 27.0*S.x - 162.0*S.y



Resultando:

\begin{equation*}
\bullet \quad \sum \vec{M}_A
= \sum_{i} (\vec{r}_{\textrm{A}i} \times \vec{F}_{i}) = \vec{0}
\end{equation*}

\begin{equation*}
\begin{aligned}
~\Leftrightarrow~ &
(0.1F-0.1F'+45.0-27.0)\,\vec{\imath} \\
&+ (-0.290F-0.3F'-0.6R_\textrm{Cz}-270.0- 162.0)\,\vec{\jmath} \\
&+ (-0.078F+0.6R_\textrm{Cy})\,\vec{k}
= \vec{0}
\end{aligned}
\end{equation*}

In [69]:
expMA = CP1 + CP2 + CP3 + CP4 + CP5
print('Exp MA   = ',expMA)

Exp MA   =  (0.1*f1 - 0.1*f2 + 18.0)*S.x + (-0.289777747886721*f1 - 0.3*f2 - 0.6*rcz - 432.0)*S.y + (-0.0776457135307562*f1 + 0.6*rcy)*S.z



Decompondo estas quantidades vetoriais nas equações escalares do equilíbrio dos momentos produzidos pelas forças externas aplicadas resulta o seguinte sistema de equações:

In [70]:
expMAi = expMA.args[0][0].args[2][0]
print('Exp MA.i = ',expMAi)

expMAj = expMA.args[0][0].args[2][1]
print('Exp MA.j = ',expMAj)

expMAk = expMA.args[0][0].args[2][2]
print('Exp MA.k = ',expMAk)

Exp MA.i =  0.1*f1 - 0.1*f2 + 18.0
Exp MA.j =  -0.289777747886721*f1 - 0.3*f2 - 0.6*rcz - 432.0
Exp MA.k =  -0.0776457135307562*f1 + 0.6*rcy



Resulta finalmente o seguinte sistema de equações (6 equações a 6 incógnitas):

\begin{equation*}
\bullet\quad
\begin{cases}
\sum F_x = 0 \\
\sum F_y = 0 \\
\sum F_z = 0 \\
\sum M_{Ax} = 0 \\
\sum M_{Ay} = 0 \\
\sum M_{Az} = 0
\end{cases}
~\Leftrightarrow~
\begin{cases}
R_\textrm{Ax} = 0 \\
R_\textrm{Ay} + R_\textrm{Cy} - F\sin(15\frac{\pi}{180}) = 0 \\
R_\textrm{Cz} + 490 + F\cos(15\frac{\pi}{180}) + F'=
0\\
0.1F-0.1F'+18.0 = 0 \\
-0.290F-0.3F'-0.6R_\textrm{Cz}-432.0 = 0 \\
-0.078F+0.6R_\textrm{Cy} = 0
\end{cases}
\end{equation*}

In [71]:
EFX = FX
EFY = FY
EFZ = FZ
EMX = expMAi
EMY = expMAj
EMZ = expMAk

sol = solve({EFX,EFY,EFZ,EMX,EMY,EMZ},{rax,ray,rcy,rcz,f1,f2})

RAx = float(sol[rax])
print('RAx = ','{:.1f}'.format(RAx),' N')
RAy = float(sol[ray])
print('RAy = ','{:.1f}'.format(RAy),' N')
RCy = float(sol[rcy])
print('RCy = ','{:.1f}'.format(RCy),' N')
RCz = float(sol[rcz])
print('RCz = ','{:.1f}'.format(RCz),' N')
F1 = float(sol[f1]); print('F(1) = ','{:.1f}'.format(F1),' N')
F2 = float(sol[f2]); print('F(2) = ','{:.1f}'.format(F2),' N')

RAx =  0.0  N
RAy =  18.4  N
RCy =  18.4  N
RCz =  -950.0  N
F(1) =  142.4  N
F(2) =  322.4  N



Numa outra abordagem, o sistema de equações do equilíbrio estático do mecanismo pode ser escrito na forma matricial seguinte:

\begin{equation*}
[A]\{x\} = \{b\}
\Leftrightarrow
\begin{bmatrix}
1& 1& 0& -\sin\alpha& 0 \\
0& 0& 1& \cos\alpha& 1 \\
0& 0& 0& 0.1& -0.1 \\
0& 0& -0.6& -0.29& -0.3 \\
0 & 0.6& 0& -0.078& 0
\end{bmatrix}
\begin{Bmatrix}
R_\textrm{Ay} \\
R_\textrm{Cy} \\
R_\textrm{Cz} \\
F \\
F'
\end{Bmatrix} =
\begin{Bmatrix}
0 \\ -490 \\ -18.0 \\ 432.0 \\  0
\end{Bmatrix}
\end{equation*}

Cuja solução é obtida multiplicando, ambos os membros, à esquerda por:
$\{x\}=[A]^{-1}\{b\}$.

In [72]:
np. set_printoptions(precision=1, suppress=True)

A2 = [1, 1, 0, -np.sin(angB), 0]
A3 = [0, 0, 1, np.cos(angB), 1]
A4 = [0, 0, 0, 0.1, -0.1]
A5 = [0, 0, -0.6, -0.29, -0.3]
A6 = [0, 0.6, 0, -0.078, 0]

A = np.array([A2,A3,A4,A5,A6])
print(f'A =\n {A}')

invA = np.linalg.inv(A)
print(f'A^-1 =\n {invA}')

b = np.array([0,-490,-18.0,432.0,0])
print(f'b = {b}')

x = np.linalg.solve(A,b)
print(f'x = {x} [N / N.mm]')

A =
 [[ 1.   1.   0.  -0.3  0. ]
 [ 0.   0.   1.   1.   1. ]
 [ 0.   0.   0.   0.1 -0.1]
 [ 0.   0.  -0.6 -0.3 -0.3]
 [ 0.   0.6  0.  -0.1  0. ]]
A^-1 =
 [[ 1.   0.1  0.7  0.2 -1.7]
 [ 0.   0.1  0.7  0.2  1.7]
 [ 0.  -1.  -0.  -3.3  0. ]
 [ 0.   1.   5.1  1.7  0. ]
 [-0.   1.  -4.9  1.7 -0. ]]
b = [   0. -490.  -18.  432.    0.]
x = [  18.4   18.5 -950.1  142.5  322.5] [N / N.mm]



O sentido negativo para $R_{Cy}$ indica simplesmente que a direção escolhida para o sentido positivo no DCL é, na verdade, contrário ao assumido, i.e., $R_{Cy}$ atua no sentido contrário ao sentido positivo para o eixo $z$.

#### c) As reações nas duas chumaceiras. <a id="C"></a> ([_enunciatum_](#P7))

\begin{equation*}
\begin{cases}
R_\textrm{Ax} = 0~\textrm{N} \\
R_\textrm{Ay} = 18.4~\textrm{N}\\
R_\textrm{Az} = 10~\textrm{N}\\
R_\textrm{Cy} = 18.4~\textrm{N}\\
R_\textrm{Cz} = -950.0~\textrm{N}
\end{cases}
\end{equation*}


**Em resumo**

Da análise do equilíbrio estático foi possível determinar a intensidades de todas as forças identificadas e representadas no DCL:

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/Notebooks/Ch04/P6/MAI_Ch04_P6d.jpg"
width="500">



### Apêndice

Qualquer sistema de forças, $F_1, F_2, \ldots, F_n$ pode ser substituído por um sistema força-binário equivalente aplicado num dado ponto $O$, constituído pelos vetores $\vec{R}$ e $M_0^R$.

Quando as forças que actuam sobre um corpo podem ser reduzidas a um sistema força-binário constituído por uma força nula e um binário nulo, diz-se que o corpo está em equilíbrio estático.

As condições necessárias e suficientes para garantir o equilíbrio são:

\begin{equation*}
\begin{cases}
\sum\vec{F} = \vec{0}\\
\sum\vec{M}_0 = \sum\vec{r}\times\vec{F} = \vec{0}
\end{cases}
\end{equation*}

Decompondo a força e o momento nas suas componentes cartesianas, num sistema de coordenadas ortonormado e direto, é possível escrever as seguintes seis equações escalares:

\begin{equation*}
\begin{cases}
~\sum F_x = 0\\
~\sum F_y = 0\\
~\sum F_z = 0\\
\sum M_{0x} = 0 \\
\sum M_{0y} = 0 \\
\sum M_{0z} = 0
\end{cases}
\end{equation*}

Para escrever as equações de equilíbrio de um corpo rígido é essencial identificar todas as forças aplicadas no corpo.  Essas forças devem constar do diagrama de corpo livre.

**Categorias de equações de equilíbrio no plano (3D)**

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/Notebooks/GFormulas/EQ3D.jpg"
width="500">

**Reações nos apoios em três dimensões**

As reações sobre um corpo rígido estrutural tridimensional podem ir desde uma única força de direção e sentido desconhecida exercida por uma superfície lisa, a um sistema força-binário, exercido por um encastramento. Pelo que o número de incognitas aasocidas a uma reação pode varia de um a seis nos problemas que envolvem o equilíbrio estático geral. A figura infra sumaria vários tipos de ligações e correspondentes reações. Uma força simples de determinar o tipo de reação e consequentemente o número de incógnitas envolvidas, é verificar quais dos seis graus de liberdade (movimentos fundamentais de translação nas direções $x$, $y$ e $z$ e rotação em torno dos eixos $x$, $y$ e $z$) são permitidos e quais os impedidos.

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MAI/Notebooks/GFormulas/ReacoesApoios3D.jpg"
width="500">

É de notar que alguns apoios (e.g. dobradiças, chumaceira) possuem reações em termos de forças e momentos. No entanto quando estes apoios são usados conjuntamente no corpo rígido com outros apoios iguais, devidamente alinhados, então as forças de reação destes apoios, sozinhas, são suficientes para manter o corpo em equilíbrio. Ou seja, os momentos tornam-se redundantes e devem, por isso, ser omitidos do diagrama de corpo livre, possibilitando a resolução do problema com a análise das equações de equilíbrio.


---

Copyright (c) DEMI - FCT NOVA

Interactive computing by <a href="https://jupyter.org/" target="_blank"> <span
style="color:#333399"> Jupyter Notebook </span> </a> &nbsp;|&nbsp;Coded by <a href = "mailto: jmc.xavier@fct.unl.pt">José Xavier</a>

Licensed under  <a href="http://creativecommons.org/licenses/by-sa/4.0/"
target="_blank"> <span style="color:#333399;font-size: 20px"> CC BY-SA 4.0  </span></a>